src/main/java/nl/mtvehicles/core/movement/VehicleMovement.java

```diff
@@ -342,14 +341,6 @@ protected boolean blockCheck() {
         final BlockData blockData = loc.getBlock().getBlockData();
         final BlockData blockDataBelow = locBlockBelow.getBlock().getBlockData();
 
-        if (vehicleType.isBoat()){
-            if (!locBlockBelow.getBlock().getType().toString().contains("WATER")){
-                VehicleData.speed.put(license, 0.0);
-                return false;
-            }
-
-            return false;
-        }
```

Comment:
I see you've changed this in `VehicleMovement`, but left it in `VehicleMovement1_12` - is that intentional?

Reply:
Kelp and other aquatic plants were not added until 1.13, so that was intentional, yes.

Reply:
Oh, tru dat XD